### PR TITLE
Updated API docs for 0.9.1909.3002.

### PR DESCRIPTION
**Pull request by cgranade**
This PR updates API documentation based on release 0.9.1909.3002.

### CODE REVIEW

File: api/qsharp/microsoft.quantum.logical.xor.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+### YamlMime:QSharpType
+uid: microsoft.quantum.logical.xor
+name: Xor
+type: function
+namespace: Microsoft.Quantum.Logical
+summary: Returns the Boolean exclusive disjunction of two values.
+syntax: 'function Xor (a : Bool, b : Bool) : Bool'
```
Comment (bettinaheim) [inline api/qsharp/microsoft.quantum.logical.xor.yml]: This one opposed to all other docs does not seem to have a remark with the ^^^ syntax.How come?

Comment (cgranade) [reply api/qsharp/microsoft.quantum.logical.xor.yml]: I thought `^^^` was bitwise, not Boolean?

Comment (bettinaheim) [reply api/qsharp/microsoft.quantum.logical.xor.yml]: Ah, sorry, yes, did not read carefully enough...

**Comment by cgranade** [comment]
Thanks, @bettinaheim!

**Pull request merged**